backend/src/middleware/hashtags/extractHashtags.js

```diff
@@ -6,21 +6,21 @@ import { exec, build } from 'xregexp/xregexp-all.js'
 //    0. Search for whole string.
 //    1. Hashtag has only all unicode characters and '0-9'.
 //    2. If it starts with a digit '0-9' than a unicode character has to follow.
-const regX = build('^/search/hashtag/((\\pL+[\\pL0-9]*)|([0-9]+\\pL+[\\pL0-9]*))$')
+const regX = build('^((\\pL+[\\pL0-9]*)|([0-9]+\\pL+[\\pL0-9]*))$')
 
 export default function(content) {
   if (!content) return []
   const $ = cheerio.load(content)
   // We can not search for class '.hashtag', because the classes are removed at the 'xss' middleware.
   //   But we have to know, which Hashtags are removed from the content as well, so we search for the 'a' html-tag.
-  const urls = $('a')
+  const ids = $('a[data-hashtag-id]')
```

Comment:
I like this change!

Reply:
@mattwr18 This change breaks the functionality. 😂